doc/tutorial_ti_cc3200.md

```diff
@@ -105,152 +105,112 @@ _You should have a Xively account already created, but if you do not, register o
+#### Code Composer Studio project configuration
 
-Reaching this point means you are able to produce and execute CC3200 compatible binary on the device itself.  Congratulations!
-
-**NOTE**: As per Texas Instruments instructions, keep the J15 Jumper set to ON and push Reset button on the board before each debug session. In case of trouble review the [TI's CC3200 help doc](http://www.ti.com/lit/ds/symlink/cc3200.pdf)
+In order to adapt the imported project to your environment you have to modify project variables that describes two very important locations on your hard drive.
 
-### Add the Xively Client to ent_wlan
+1. In Code Composer Studio™, make sure the ```xively_demo``` project is highlighted
+2. Select ```Project```->```Properties```
+3. Highlight```Resource```->```Linked Resources```
+4. Double click on ```CC3200_SDK_ROOT``` variable name and using the ```Folder``` button navigate to where your ```CC3200_SDK``` is installed, mark the subfolder ```cc3200-sdk``` in SDK's main directory and hit ```OK``` twice.
+5. Double click on ```XIVELY_LIBRARY_C_ROOT``` variable name and again using the ```Folder``` make the variable to point to the ```xively-client-c``` folder.
 
-Next we're going to add a function to connect to the Xively Broker. Its implementation is based on the examples in the Client repo, e.g. `xively-client-c/examples/mqtt_logic_producer/src/mqtt_logic_producer.c`.
+#### xively_demo configuration
 
-- Paste the following code within `main.c` of the _ent_wlan_ anywhere in the main portion of the file _before_ the location where we will call the function, which will be around line 647 (see the following steps).
+Before you build your application you will have to set some data like your WiFi router credentials and Xively device credentials.  
 
-        #include <xively.h>
-        #include <stdio.h>
+- update wifi settings in main.c
 
-        void on_connection_state_changed( xi_context_handle_t in_context_handle,
-                                          void* data,
-                                          xi_state_t state )
-        {
-            printf( "Hello Xively World!, state: %d\n", state );
-        }
+    - update AP name and password defines according to your wifi settings:
 
-        void ConnectToXively()
-        {
-            xi_initialize( "xi_account_id", "xi_device_id", 0 );
+            #define ENT_NAME    "AccessPointName"
+            #define USER_NAME   "UsernameIfAny"
+            #define PASSWORD    "Password"
 
-            xi_context_handle_t xi_context = xi_create_context();
+    - select a security type in the `MainLogic()` function according to your wifi settings.  For example, in the case of WPA2 set
 
-            if ( XI_INVALID_CONTEXT_HANDLE >= xi_context )
-            {
-                printf( " xi failed to create context, error: %d\n", xi_context );
-            }
+            g_SecParams.Type = SL_SEC_TYPE_WPA_WPA2;
```

Comment:
we should tell them where to find the enumerate types otherwise they won't know what to select from.